.gitlab/docker/Dockerfile.spack_build

```diff
@@ -11,8 +11,8 @@ ARG NUM_PROCS
 
 COPY . ${SOURCE_DIR}
 
-# Print spack spec since not printed if the compiler image is found on jfrog
-RUN spack spec -lI $spack_spec
+## Print spack spec since not printed if the compiler image is found on jfrog
+#RUN spack spec -lI $spack_spec
```

Comment:
Please uncomment this again (and add the deprecated flag to the spack config instead of passing it as a flag to `install`).